webpack.config.js

```diff
@@ -3,6 +3,7 @@ const path = require('path');
 const fs = require('fs');
 const glob = require('glob');
 const _ = require('lodash');
+const webpack = require('webpack'); // eslint-disable-line
```

Comment:
What rule is disabled here ?

Reply:
`import/no-extraneous-dependencies` due to the fact that webpack isn't defined as a dependency from the `ovh-manager-cloud` because it came from the shared configuration (https://github.com/ovh-ux/manager-webpack-config/blob/master/package.json#L58)